cmd/kubelet/app/server.go

```diff
@@ -211,6 +212,7 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.ResourceContainer, "resource-container", s.ResourceContainer, "Absolute name of the resource-only container to create and run the Kubelet in (Default: /kubelet).")
 	fs.StringVar(&s.CgroupRoot, "cgroup_root", s.CgroupRoot, "Optional root cgroup to use for pods. This is handled by the container runtime on a best effort basis. Default: '', which means use the container runtime default.")
 	fs.StringVar(&s.ContainerRuntime, "container_runtime", s.ContainerRuntime, "The container runtime to use. Possible values: 'docker', 'rkt'. Default: 'docker'.")
+	fs.IntVar(&s.MaxPods, "max_pods", 32, "Number of Pods that can run on this Kubelet.")
```

Comment:
Wojtek and Filip believe that 30 should be enough. Only place where higher number appears is in Density test, where we allow 50 and 100 Pods per Node, but those are disabled by default. I changed to 100, but I'm not really convinced that a good call.